SU-SWS/decanter
# D8CORE-105: Local footer.

# READY FOR REVIEW

# Summary
- Creates a responsive local footer component
- Adds FA assets 
- Adds a link list macro
- Adds signup form component
- Updates Dependencies
- Adds tugboat

# Review By (Date)
- Please review by October 18

# Urgency
- Med-high as we would like to get it deployed to our pilot at the end of the month

# Steps to Test

1. Check out this branch
2. Update npm dependencies `npm install`
3. Compile code `npm run build`
4. Compile styleguide `npm run styleguide`
5. Review code
6. Compare to mock: https://www.figma.com/file/Kmd4utmJFPRMVeCFEEBQhLtx/Stanford-Design-Library?node-id=5268%3A11637
7. Reivew for accessibility

# Affected Projects or Products
- Decanter
- D8 Core project

# Associated Issues and/or People
- https://github.com/SU-SWS/decanter/issues/491
- @kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
